tractor/_trionics.py

```diff
@@ -297,3 +352,8 @@ def do_hard_kill(proc):
     # TODO: figure out supervisors from erlang
     async with ActorNursery(current_actor()) as nursery:
         yield nursery
+
+
+def is_main_process():
+    "Bool determining if this actor is running in the top-most process."
```

Comment:
Technically, might want to change this to triple quotes to pass future style checkers.

Reply:
Thanks